integration/ksm/ksm_test.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -e
+
+dir_path=$(dirname "$0")
+source "${dir_path}/../../lib/common.bash"
+source "${dir_path}/../../metrics/lib/common.bash"
+
+TEST_NAME="Run containers with KSM"
+KATA_KSM_THROTTLER="${KATA_KSM_THROTTLER:-yes}"
+RUNTIME="${RUNTIME:-kata-runtime}"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
+IMAGE="${IMAGE:-busybox}"
+WAIT_TIME="60"
+
+function turn_on_ksm() {
+	echo "Turning on KSM"
```

Comment:
so, not going to use the existing functions in the library code then? - the save/enable_aggresive/restore ? That has a few benefits:
- it puts KSM back how it found it
- setting the KSM aggresive makes the test settle down much quicker, particularly on ubuntu that has terrible default ksm settings....